test/CMakeLists.txt

```diff
@@ -60,27 +60,27 @@ function(cub_add_test target_name_var test_name test_src cub_target)
   )
 endfunction()
 
-# Sets HAS_QUICK_VARIANT / HAS_QUICKER_VARIANT / NO_VARIANTS to True/False in
+# Sets HAS_BENCHMARK_VARIANT / HAS_MINIMAL_VARIANT / NO_VARIANTS to True/False in
 # the calling scope.
 # Used to detect variants of unit tests depending on whether a source file
-# contains the strings "QUICK_TEST" or "QUICKER_TEST".
+# contains the strings "BENCHMARK" or "MINIMAL_TEST".
```

Comment:
These should check for the full strings `CUB_TEST_BENCHMARK`, `CUB_TEST_MINIMAL` to avoid false positives.

Checking for `MINIMAL_TEST` won't match the new `#ifdef`s (`CUB_TEST_MINIMAL`).

Reply:
Good catch.